src/backend/base/langflow/components/processing/converter.py

```
@@ -0,0 +1,138 @@
+        ),
+        TabInput(
+            name="output_type",
+            display_name="Output Type",
+            options=["Message", "Data", "DataFrame"],
+            info="Select the desired output data type",
+            real_time_refresh=True,
+            value="Message",
+        ),
+    ]
+
+    outputs = [Output(display_name="Message Output", name="message_output", method="convert_to_message")]
```

Pylint (3.3.7)

[convention] Line too long (105/100)

(C0301)